torchvision/ops/stochastic_depth.py

```diff
@@ -0,0 +1,56 @@
+import torch
+from torch import nn, Tensor
+
+
+def stochastic_depth(input: Tensor, mode: str, p: float, training: bool = True) -> Tensor:
```

Comment:
What do you think about making the signature be
```python
input: Tensor, p: float, mode: str, training: bool = True
```
instead? I think having `p` come first might be more natural?